ReSharper.FSharp/test/src/FSharp.Tests.Host/FSharpTestHost.fs

```diff
@@ -41,12 +44,26 @@ type FSharpTestHost
         |> Option.map (fun options ->
             options.OtherOptions
             |> Array.choose (fun o -> if o.StartsWith("-r:") then Some (o.Substring("-r:".Length)) else None)
-            |> Array.map (fun s -> FileSystemPath.TryParse(s))
+            |> Array.map (FileSystemPath.TryParse)
             |> Array.filter (fun p -> not p.IsEmpty && directory.IsPrefixOf(p))
             |> Array.map (fun p -> p.Name)
             |> List)
         |> Option.defaultWith (fun _ -> List())
 
+    let setFeatures (rdFeatures: RdSetFeatures) =
```

Comment:
The name doesn't imply enabling/disabling at all, could we have a better one? please?